api/src/main/java/org/openmrs/module/ModuleClassLoader.java

```diff
@@ -276,9 +274,7 @@ private static URL getClassBaseUrl(final Class<?> cls) {
 		
 		// add each defined jar in the /lib folder, add as a url in the classpath of the classloader
 		try {
-			if (log.isDebugEnabled()) {
-				log.debug("Expanding /lib folder in module");
-			}
+				log.debug("Expanding {} /lib folder in module");
```

Comment:
There is no variable in the log message `log.debug("Expanding /lib folder in module");`

It will always log `"Expanding /lib folder in module"`.

So you do not need to add the `{}`. The `{}` is a placeholder that will be replaced with the variables you pass to the log method. If there is nothing variable about the message we do not need the `{}`.